src/dlgTriggerEditor.cpp

```diff
@@ -85,6 +85,13 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 , mCleanResetQueued(false)
 , mSavingAs(false)
 , mAutosaveInterval{}
+, mTriggerEditorSplitterState(QByteArray())
```

Comment:
All of these can use the modern `{}` - and it'll pick the right datatype for you.

```suggestion
, mTriggerEditorSplitterState{}
```